applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -1866,6 +1871,10 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     {
         [self isInitialized: result];
     }
+    else if ( [@"getMaxConfiguration" isEqualToString: call.method] )
```

Comment:
I would just call this `getConfiguration`, no need for `Max` since the context is already MAX (the plugin).